### ISSUE TITLE
Mooprox doesn't start when there is no config file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fluffynuts/mooprox#1]
```
What steps will reproduce the problem?
1. Download mooprox.exe
2. double-click to run
3. tray icon appears but after a while disappears. Application doesn't respond 
to inputs

What is the expected output? What do you see instead?
I should see an empty proxy list (or a list with the system current proxy) and 
a Direct and Exit option on the systray menu as well as opening the main form 
with a click -- I don't see any of that

Workaround: Create the file %USERPROFILE%\mooprox.ini and the app will start 
and work. This will be fixed soon!


```

Original issue reported on code.google.com by `dav...@gmail.com` on 8 Dec 2011 at 4:39

**Comment by fluffynuts** [comment]
fixed in cef5e9282cc98c42b056ae352cda806f31574efd

**Comment by fluffynuts** [comment]
fixed in cef5e9282cc98c42b056ae352cda806f31574efd
